ConnorAndersonLarson/PlanetParty
# Refactor/file separation

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Places interfaces into separate file to be called externally from components
Creates API file to hand fetch requests
Cleans GET request data to only use data we need

### What ticket(s) do the changes resolve?
Close #18 
Close #19 

Please Link Issues to this PR as necessary.
#18 
#19 

### Notes for the Reviewer
N/A

### How to Test Changes?
Pull Down changes, open via `npm start`. Site should display with 8 planets (all same image) and unique names. Checking App state in React Dev Tools should show all planet objects inside state.


### Next Iterations
Translate apiCalls.js into ts, utilize interface.
